dev.md

```diff
@@ -4,6 +4,7 @@
 
 These steps are tested for Ubuntu LTS 18. 
 Other linux os or macos uses different package manager such as homebrew.
+run build.sh for automate setup, build.sh is a script that runs below steps
 
 ### Install CMake
```

Comment:
do we want to delete all these steps which is already automated in build.sh? 
If we don't want to delete it, we can move them to another .md or bottom of the file? so it looks easier for readers as they just need to know to run the build.sh

Reply:
sure, I moved the manual steps to the bottom of this file, and just listed `build.sh` and `fix grpc` those 2 steps here